src/views/home.hbs

```diff
@@ -1,4 +1,28 @@
-<div> 
-    Hello from home page
+<div id="proposal">
+    <div>
+        Create new post:
+    </div>
+    <textarea onfocus="append()" onblur="remove()" id="text-area" placeholder="Write your proposal..."></textarea>
+    <button id="post">Post</button>
+        <div id="requirements" style="display:none">
+        <ul id="spcialization">
+        <h4>specialization</h4>
+       <input id="specialization" type="text" placeholder="Enter required specialization.."></input>
```

Comment:
@Anies12 It would be really cool if these text fields for specialization and skills were instead `<select>` dropdowns, filled with all of the database's specializations and skills.